run-poc/alpha-pages/test-mi-pairs.ipw

```diff
@@ -0,0 +1,76 @@
+###LEFT-WALL### A mom is a human .
+0 ###LEFT-WALL### 1 A 0.1
```

Comment:
this looks like some random data file. Did you mean to check it in?

Reply:
It's a test file for the parsing method where MI is obtained from file (for example, where word instance-pair weights are coming from a DNN).

Reply:
Move it to a test directory?

Reply:
The purpose of this file is not different from the other ones in the same folder: it allows to easily test the integration of the different components of the MST-parser in the "file mode" and produce parses from it. The Cmake tests are unit tests checking the funcionality of the modules, but there's no automated integration test.

Reply:
The purpose of this file is not different from the other ones in the same folder: it allows to easily test the integration of the different components of the MST-parser in the "file mode" and produce parses from it. The Cmake tests are unit tests checking the functionality of the modules, but there's no automated integration test.